### PR TITLE
improved duckdb integration doc

### LINKED ISSUE #174: improve duckdb integration docs
improve [duckdb's integration](https://jupysql.ploomber.io/en/latest/integrations/duckdb.html) documentation by showing how to pass parameters to the connection: 

https://github.com/Mause/duckdb_engine#configuration

https://jupysql.ploomber.io/en/latest/howto.html#connect-to-existing-engine

### PR DESCRIPTION
**Pull request by ayushmaanshrotriya**
## Describe your changes
passed parameters into connections

## Issue ticket number and link
Closes #174 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added thorough [tests](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#testing) (when necessary).
- [x] I have added the right documentation in the [docstring](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#documenting-changes-and-new-features) and [changelog](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#changelog) (when needed)

----
:books: Documentation preview :books:: https://jupysql--190.org.readthedocs.build/en/190/

### CODE REVIEW

**Comment by ayushmaanshrotriya** [comment]
am not able to understand why the build failed.

**Comment by ayushmaanshrotriya** [comment]
#174

**Comment by edublancas** [comment]
check out the logs: https://readthedocs.org/projects/jupysql/builds/19648555/

looks like you need to install the extension first:

```

[0;31mIOException[0m: IO Error: Extension "/home/docs/.duckdb/extensions/v0.7.1/linux_amd64_gcc4/httpfs.duckdb_extension" not found.
Extension "https" is an existing extension.

Install it first using "INSTALL https".
```

let's pass some args that don't require an extension

File: doc/integrations/duckdb.md [hunk 1]
```diff
@@ -249,3 +249,21 @@ SELECT *
...
 WHERE x > 95
 ```
+
+## Passing parameters to connection
+```{code-cell} ipython3
+from sqlalchemy import create_engine
+some_engine = create_engine(
+    'duckdb:///:memory:',
+    connect_args={
+        'preload_extensions': ['https'],
```
Comment (Mause) [inline doc/integrations/duckdb.md:259]:
```suggestion
        'preload_extensions': ['httpfs'],
```

**Review by Mause — CHANGES_REQUESTED** [review]
The issue is a typo in the extension name

File: doc/integrations/duckdb.md (hunk 1 above)
Comment (ayushmaanshrotriya) [reply doc/integrations/duckdb.md:259]: thanks
am closing this pull request and creating one again

Comment (edublancas) [reply doc/integrations/duckdb.md:259]: you can push to the same branch and it'll update the pull request

Comment (Mause) [reply doc/integrations/duckdb.md:259]: Not sure why you changed it back, httpfs is the correct name - though you will need to install it first

**Comment by ayushmaanshrotriya** [comment]
finally. gotta go through the whole document about duckdb. thanks for help. review required.

**Pull request merged**